tests/unit/models/sec10k/utils_test.py

```diff
@@ -18,20 +17,23 @@
 def test_archive():
     """Return test GCSArchive class."""
     with (
-        unittest.mock.patch("mozilla_sec_eia.utils.cloud.GCSArchive._get_engine"),
-        unittest.mock.patch("mozilla_sec_eia.utils.cloud.GCSArchive._get_bucket"),
+        unittest.mock.patch(
+            "mozilla_sec_eia.models.sec10k.utils.cloud.GCSArchive._get_engine"
+        ),
+        unittest.mock.patch(
+            "mozilla_sec_eia.models.sec10k.utils.cloud.GCSArchive._get_bucket"
+        ),
     ):
-        return GCSArchive(
-            settings=GoogleCloudSettings(
-                GCS_FILINGS_BUCKET_NAME="filings_bucket_name",
-                GCS_LABELS_BUCKET_NAME="labels_bucket_name",
-                GCS_METADATA_DB_INSTANCE_CONNECTION="metadata_db_instance_connection",
-                GCS_IAM_USER="user",
-                GCS_METADATA_DB_NAME="metadata_db_name",
-                GCS_PROJECT="project_name",
-                MLFLOW_TRACKING_URI="http://tracking.server",
-            )
+        archive = GCSArchive(
+            filings_bucket_name="filings_bucket_name",
+            labels_bucket_name="labels_bucket_name",
+            metadata_db_instance_connection="metadata_db_instance_connection",
+            user="user",
+            metadata_db_name="metadata_db_name",
+            project="project_name",
         )
+        archive.setup_for_execution("fake_context")
```

Comment:
What's the test assertion here?

Reply:
This is just setting up a fixture that creates a dagster resource that gets used by other tests